packages/hebao_v1/contracts/modules/stores/WhitelistStore.sol

```diff
@@ -41,6 +48,7 @@ contract WhitelistStore is DataStore
         onlyManager
     {
         addAddressToSet(walletKey(wallet), addr, true);
+        timestampMap[wallet][addr] = now + delayToDffective;
```

Comment:
Maybe it's a good idea to just let the modules calculate the timestamp the address is actually effective? This way there's no need to configure this variable in this storage contract and modules could use extra logic because they have more context: something like the more signatures that were used to whitelist an address the less time it takes to make it effective.